pkg/monitor/dynkeepalived.go

```diff
@@ -98,10 +98,15 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 	var appliedConfig, curConfig, prevConfig *config.Node
 	var configChangeCtr uint8 = 0
 
+	clusterName, _, err := config.GetClusterNameAndDomain(kubeconfigPath, clusterConfigPath)
+	if err != nil {
+		return err
+	}
+
 	if exists, err := needLease(cfgPath); err != nil {
```

Comment:
Need lease might be on a specific VIP.
User might choose to supply IP for ingress but use dhcp for API

Need to decide if supporting this is critical for now

Reply:
I suggested to @romfreiman that for now we just use this dumb condition instead of defining how the file would look like.
Do you think it is critical?

Reply:
Will be defined later.